submissions/zhenyakornilov/html-css-popup/index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./styles.css">
+  <title>PopUp</title>
+</head>
+
+<body>
+  <header class="header">
+    <nav class="navbar">
+      <div class="nav-logo image-wrapper">
+        <span class="nav-logo-text">HTML & CSS</span>
+        <span><img style="margin-left: 5px" src="./images/favicon.png" alt="kitten-icon"></span>
+      </div>
+      <ul class="nav-links">
+        <div class=""></div>
+        <li class="nav-item">
+          <a class='link' href="#">Mail</a>
+        </li>
+        <li class="nav-item">
+          <a class='link' href="#">Images</a>
+        </li>
+
+        <!-- li contains the whole popup window -->
+        <li class="nav-item">
+          <input class="popup-toggle" type="checkbox" id="check-toggle">
+          <label class="label-toggle" for="check-toggle">
+            <img class='popup-img' src="./images/popup-button.png" alt="popup-button">
+          </label>
+          <div class="popup">
```

Comment:
Semantically, here is also a list

Reply:
Still not fixed. You need change tags here to make this list semantically correct